generator/lib/code_builders/assignment_builder.dart

```diff
@@ -17,124 +17,122 @@ import 'package:smartstruct_generator/models/source_assignment.dart';
 /// fromSub(model.sub);
 /// ```
 ///
-Expression generateSourceFieldAssignment(SourceAssignment sourceAssignment,
-    ClassElement abstractMapper, VariableElement targetField) {
+Expression generateSourceFieldAssignment(
+    SourceAssignment sourceAssignment, ClassElement abstractMapper, VariableElement targetField) {
```

Comment:
Hey, as already mentioned in your E-Mail (sorry for the late reply :(), would be nice if we could keep the original formatting (for now). 
I am not sure if it is possible to configure the formatter on project level, but maybe this would be a good idea to try, as this would avoid the clash of different formatters when collaborating.

Reply:
I am so sorry. I change the code formatting too much. If we can come to a consensus view about this idea. I will change the pr and keep the origin formatting.